### PR TITLE
feat(sign-off): Map signoff and add edit signoff for item(SDNTB-24, S…

### PR DESCRIPTION
**Pull request by ioanpocol**
…D-NTB-25)

Those changes are available only if sign_off mapping is set on
settings.py and index.html
When an user is added/edited, the sign-off is not shown and is set to
mapped value
On authoring the user is able to set a new value for sign-off by
selecting only one user from user list

### CODE REVIEW

File: apps/archive/common.py [hunk 1]
```diff
@@ -186,6 +186,7 @@ def set_sign_off(updates, original=None, repo_type=ARCHIVE, user=None):
     Set sign_off on updates object. Rules:
         1. updates['sign_off'] = original['sign_off'] + sign_off of the user performing operation.
         2. If the last modified user and the user performing operation are same then sign_off shouldn't change
+        3. If sign_off is received on updates, this value will be preserved
```
Comment (akintolga) [inline apps/archive/common.py]: A new behave test in archive.feature would be good for this behavior.

**Comment by akintolga** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

File: apps/archive/common.py (hunk 1 above)
Comment (ioanpocol) [inline apps/archive/common.py]: Done

**Pull request merged**